org.civicrm.bootstrapcivicrm/scss/bootstrap/bootstrap.scss

```diff
@@ -5,14 +5,14 @@
 @import "mixins/*";
 
 #{$bootstrap-namespace} {
-    @import "vendor/text-angular/text-angular";
-    @import "vendor/ui-bootstrap/ui-bootstrap";
-    @import "vendor/ui-select/ui-select";
+  @import "vendor/ui-bootstrap/ui-bootstrap";
+  @import "vendor/ui-select/ui-select";
 
-    @import "overrides/style/*";
-    @import "components/*";
+  @import "overrides/style/*";
+  @import "components/*";
 }
 
+@import "vendor/text-angular/text-angular";
```

Comment:
Please check it in the Appraisals extensions as well (you probably need to install the extension on your local site). I believe text-angular is used in one of the modals